internal/plugins/helm/v1/scaffolds/init.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scaffolds
+
+import (
+	"fmt"
+	"os"
+
+	"sigs.k8s.io/kubebuilder/pkg/model"
+	"sigs.k8s.io/kubebuilder/pkg/model/config"
+	"sigs.k8s.io/kubebuilder/pkg/plugin/scaffold"
+
+	"github.com/operator-framework/operator-sdk/internal/kubebuilder/machinery"
+	"github.com/operator-framework/operator-sdk/internal/plugins/helm/v1/chartutil"
+	"github.com/operator-framework/operator-sdk/internal/plugins/helm/v1/scaffolds/templates"
+	"github.com/operator-framework/operator-sdk/internal/plugins/helm/v1/scaffolds/templates/manager"
+	"github.com/operator-framework/operator-sdk/internal/plugins/helm/v1/scaffolds/templates/metricsauth"
+	"github.com/operator-framework/operator-sdk/internal/plugins/helm/v1/scaffolds/templates/prometheus"
+)
+
+const (
+	// KustomizeVersion is the kubernetes-sigs/kustomize version to be used in the project
+	KustomizeVersion = "v3.5.4"
+
```

Comment:
Add a `HelmOperatorVersion` variable so we can download the proper version in the Makefile? We should extract the method we use in the Dockerfile template to use that logic for both the Dockerfile and the Makefile.
```go
var HelmOperatorVersion = strings.TrimSuffix(version.Version, "+git")
```

Reply:
done